Build failure on finding NETStandard.Library

### Describe the Bug
Using `Invoke-Build Test` fails with message about NETStandard.Library.

### Steps To Reproduce
```
PS C:\src\Pode> Invoke-Build Test
Build Test C:\src\Pode\pode.build.ps1
Task /Test/Build/BuildDeps
Done /Test/Build/BuildDeps 00:00:00.0092515
Task /Test/Build
MSBuild version 17.4.0+18d5aef85 for .NET
  Determining projects to restore...
C:\src\Pode\src\Listener\Pode.csproj : error NU1102: Unable to find package NETStandard.Library with version (>= 2.0.3)
C:\src\Pode\src\Listener\Pode.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 2.0.0 ]
  Failed to restore C:\src\Pode\src\Listener\Pode.csproj (in 206 ms).
ERROR: dotnet build failed for netstandard2
At C:\src\Pode\pode.build.ps1:197 char:13
+             throw "dotnet build failed for netstandard2"
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
At C:\src\Pode\pode.build.ps1:187 char:1
+ task Build BuildDeps, {
+ ~~~~~~~~~~~~~~~~~~~~~~~
Build FAILED. 3 tasks, 1 errors, 0 warnings 00:00:01.7983296
Exception: C:\src\Pode\pode.build.ps1:197
Line |
 197 |              throw "dotnet build failed for netstandard2"
     |              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | dotnet build failed for netstandard2
```
### Expected Behavior
Pode builds.

### Platform
 - OS: Microsoft Windows 10 Pro 10.0.19045
 - Browser: n/a
 - Versions:
   - Pode: 2.7.2
   - PowerShell: 7.3.0

--- a/src/Listener/NuGet.config
+++ b/src/Listener/NuGet.config
@@ -1,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    </packageSources>
+</configuration>

